### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by jameswthorne**
* Cleaned up UI
* Updated Twitter Bootstrap to v3.3.7
* Updated jquery to 1.12.4
* Added mobile view port

### CODE REVIEW

**Comment by nichochar** [comment]
This is great. Could you add some screenshots to the PR discussion so people can observe the change ? Thanks!

**Comment by jameswthorne** [comment]
Here you go!

Desktop
-------

http://public.thornelabs.net/snappass-ui-improvements-index.png
http://public.thornelabs.net/snappass-ui-improvements-share-password-link.png
http://public.thornelabs.net/snappass-ui-improvements-password.png

Responsive
----------

How it will look on mobile:

http://public.thornelabs.net/snappass-ui-improvements-index-responsive.png
http://public.thornelabs.net/snappass-ui-improvements-share-password-link-responsive.png
http://public.thornelabs.net/snappass-ui-improvements-password-responsive.png

**Comment by nichochar** [comment]
This is really great, thanks @jameswthorne 

I'm a big fan of mobile friendliness.

File: snappass/static/bootstrap/js/npm.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
+require('../../js/transition.js')
```
Comment (nichochar) [inline snappass/static/bootstrap/js/npm.js]: What is this file, and where do these dependencies enter the equation ?

File: snappass/templates/set_password.html [hunk 2]
```diff
@@ -3,27 +3,41 @@
...
+        </div>
+
+        <div class="col-sm-4">
+          <button type="submit" class="btn btn-primary" id="submit">Generate URL</button>
+        </div>
+      </form>
+    </div>
+
+    <h3>How SnapPass Works</h3>
+    <p>SnapPass allows you to share passwords with anybody in a secure, ephemeral way.</p>
```
Comment (nichochar) [inline snappass/templates/set_password.html]: I prefer the original copy, mostly because it's shorter and more to the point. Can we revert it?

**Comment by jameswthorne** [comment]
@nichochar the npm.js file comes from the latest version of Twitter Bootstrap. Not sure where the dependencies mention come in. 

Reverting to the original copy is fine by me.

**Comment by nichochar** [comment]
I would prefer to delete the dependencies if we don't need them, yes. Thanks!

**Comment by jameswthorne** [comment]
Looks like the npm.js file isn't needed at all. I deleted it.

I reverted to the original copy with some small tweaks.

**Comment by jameswthorne** [comment]
@nichochar What are you thoughts on creating an **about** page to put a more verbose version of "How SnapPass works"? That page would also be used for API documentation - when that feature gets added, of course.

**Comment by nichochar** [comment]
I like the idea of an about page much more than overloading the splash page. I think this product should be extremely easy to use, with minimal reading (ideally none, but that's a little rough)

**Comment by samueldg** [comment]
Hey @jameswthorne, FWIW I had a look at your changes and it's looking good! Glad you bumped jquery and bottstrap too.

I did notice one thing though: I think it broke the changes I had made for https://github.com/pinterest/snappass/pull/18


You can see from the screenshot that the browser keeps the passwords for autocompletion:
password _autocomplete

**Comment by jameswthorne** [comment]
Doh! I'll get that fixed tonight. Thanks for catching that.

**Pull request merged**